src/sentry/demo/tasks.py

```diff
@@ -0,0 +1,25 @@
+from sentry.models import Organization, OrganizationStatus, User
+from sentry.tasks.base import instrumented_task
+from sentry.tasks.deletion import delete_organization, retry
+from sentry.exceptions import DeleteAborted
+
+MAX_RETRIES = 5
+
+
+@instrumented_task(
+    name="sentry.demo.tasks.delete_organization_and_user",
+    queue="cleanup",
+    default_retry_delay=60 * 5,
+    max_retries=MAX_RETRIES,
+)
+@retry(exclude=(DeleteAborted,))
+def delete_organization_and_user(organization_id, user_id):
+    Organization.objects.filter(id=organization_id).update(
```

Comment:
@scefali should we wrap this in a try/except block?

Reply:
@NisanthanNanthakumar Nah, if this fails I want it to blow up and cause a Sentry error

Reply:
also, even if the org is already deleted that line won't error out (it will just do an update on 0 items).